ontology/yaml/resources/units/units.yaml

```diff
@@ -12,58 +12,319 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+acceleration:
+- meters_per_second_per_second: STANDARD
+
+adimensional:	
+- no_units: STANDARD
+
+angle:
+- degrees_angular: STANDARD
+- radians
+
+angularmomentum:
+- joule_seconds: STANDARD
+
+apparentpower:
+- volt_amperes: STANDARD
+-	kilovolt_amperes
+-	mega_volt_amperes
+
+area:
+- square_meters: STANDARD
+- square_centimeters
+- square_feet
+- square_inches
+
 concentration:
 - parts_per_million: STANDARD
-current:
+- parts_per_billion
+
+density:
+-	kilogram_per_cubic_meter: STANDARD
+
+electriccapacitance:
+-	farads: STANDARD
+
+electricconductance:
+- siemens: STANDARD
+
+electricconductivity:
+- siemens_per_meter: STANDARD
+
+electriccurrent:
 - amperes: STANDARD
-- milliamperes
+- milli_amperes
+
+electriccurrentdensity:
+- amperes_per_square_meter: STANDARD
+
+electricfield:
+-	volts_per_meter: STANDARD
+
+electricinductance:
+-	henrys: STANDARD
+
+electricphasedifference:
+-	degrees_phase: STANDARD
+
+electromagneticfieldstrength:
+-	amperes_per_meter: STANDARD
```

Comment:
spacing seems inconsistent -- why is this line (72) different than say the next one (75)?

Reply:
Thanks, it was an issue with tabs being kept after a copy/paste from the BACnet units specs. Corrected now, thanks for spotting it!